review_practice/react-app/src/components/DatePicker/style.js

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+
+export const DatePicker = styled.div`
+    && {
```

Comment:
```suggestion
    {
```

습관적으로 `&&` 셀렉터를 사용하여 스타일을 지정하는 것 같은데, 이렇게 될 경우 (예를 들어 `DatePicker` 라는 클래스명을 경우) `.DatePicker.DatePicker` 와 같이 두 번 연속 셀렉터가 정의되어 불필요하게 우선 적용 순위가 올라가게 됩니다. 정말로 우선순위를 높이는 목적이 아니라면 이러한 형태의 셀렉터는 지양할 필요가 있습니다. (`!important` 사용을 지양하자는 뜻과 유사합니다.)